packages/astro/test/units/assets/fonts/utils.test.js

```diff
@@ -179,7 +179,7 @@ describe('fonts utils', () => {
 		it('should return null if there are no fallbacks', async () => {
 			assert.equal(
 				await generateFallbacksCSS({
-					family: 'Roboto',
+					family: { name: 'Roboto' },
```

Comment:
This feels like a step down from the previous version.  Plus, the RFC still shows that it's possible to use `family: 'Roboto'`

I'm not questioning the decision, but the introduction of the `as` of this PR is in conflict with the RFC, and the RFC usually is the source of truth unless said otherwise.

Reply:
Yeah related to the other comment from Matt, this is not a public API